src/types/model.ts

```diff
@@ -0,0 +1,187 @@
+import { DuosUserResponse } from './responseTypes';
```

Comment:
I wonder if there's a tool we can use to generate these automatically. Future investigation?

Reply:
I would also imagine there are tools to generate the swagger doc from Springboot as well, so it might be a way we can make it consistent across the board.

Reply:
I think you're thinking of https://swagger.io/tools/swagger-codegen/
We are using Dropwizard on the back end, so no spring in this particular stack.